data/category-entertainment

```diff
@@ -12,6 +12,7 @@ include:dazn
 include:deviantart
 include:disney
 include:hbo
+include:hoyoverse
```

Comment:
```suggestion
include:mihoyo
```

Reply:
If I include mihoyo, It will include Chinese entertainment. The hoyoverse only have entertainment outside mainland China.